pydicom/pixel_data_handlers/gdcm_handler.py

```diff
@@ -0,0 +1,110 @@
+import sys
+have_numpy = True
+try:
+    import numpy
+except ImportError:
+    have_numpy = False
+    raise
+
+have_gdcm = True
```

Comment:
I'd rather use all capitals very sparingly, if at all.  For me, python's 'duck typing' concept means you don't have to embed information about a variable/constant into its name, including to me whether it is global or not.  Pep-8 does suggest constants in all capitals, but I personally don't need the 'shouting' feel when reading them.